tools/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/mattermost/mattermost-server/tools
```

Comment:
I'm wondering if we need jet another directory for this. We have a `build` directory with e.g. the jenkins files. What do you think about reusing this one?

Reply:
I think we should keep separate directories for `build` and `tools` because the two seem separate concerns to me, and are best kept in separate places. And more importantly, this is a separate sub-module, which means we need to be very careful about what goes in as top-level go commands will completely ignore this directory. And I am wary about adding any needless code other than tool dependencies in that.

And lastly, with 1.15, this will anyways go away and be part of the repo root.

Hence I would suggest to keep this separate for now.